include/vctr/Containers/Vector.h

```diff
@@ -257,6 +257,32 @@ class Vector : public VctrBase<ElementType, detail::StdVectorType<ElementType, A
     }
 
     //==============================================================================
+    // Accessing the underlying vector.
+    //==============================================================================
+    /** Returns a reference to the underlying storage.
+
+        The returned type is std::vector, possibly with a custom allocator type.
+     */
+    constexpr StdVectorType& getUnderlyingVector() { return Vctr::storage; }
+
+    /** Returns a reference to the underlying storage.
+
+       The returned type is std::vector, possibly with a custom allocator type.
+     */
+    constexpr const StdVectorType& getUnderlyingVector() const { return Vctr::storage; }
+
+    /** Moves the underlying storage out of this wrapper class.
+
+       The returned type is std::vector, possibly with a custom allocator type.
+     */
+    constexpr StdVectorType&& moveUnderlyingVector() && { return std::move (Vctr::storage); }
+
+    /** Shortcut to getUnderlyingVector. */
+    constexpr operator StdVectorType&() { return Vctr::storage; }
+
+    /** Shortcut to getUnderlyingVector. */
+    constexpr operator const StdVectorType&() const { return Vctr::storage; }
```

Comment:
Maybe here you could mention 'conversion operator' as a keyword in the doc for those who are searching the code to finde how it's defined

Reply:
Good point, I'll add that